packages/terra-form/tests/nightwatch/textarea-field/PopulatedTextareaField.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import TextareaField from '../../../lib/TextareaField';
+
+const textareaField = () =>
+  <TextareaField
+    label="Profile Description"
+    name="profile_description"
+    defaultVault="Hello! I'm a Software engineer!"
```

Comment:
Fixed it here. https://github.com/cerner/terra-core/pull/410/commits/a6d5d235c4e1ca12bf42865c62387b91aaa9839f